frontend/integration-tests/views/monitoring.view.ts

```diff
@@ -32,3 +33,29 @@ export const modalConfirmButton = $('#confirm-action');
 export const alertManagerYamlForm = $('.co-alert-manager-yaml__form');
 export const successAlert = $('.pf-m-success');
 export const helpText = $('.co-help-text');
+
+// Configuration Overview
+export const alertRoutingHeader = $('[data-test-section-heading="Alert Routing"]');
+export const alertRoutingEditButton = $('.co-alert-manager-config__edit-alert-routing-btn');
+export const disabledDeleteReceiverMenuItem = $(
+  '.pf-c-dropdown__menu-item.pf-m-disabled[data-test-action="Delete Receiver"]',
```

Comment:
I'd keep this as simple as possible.

```suggestion
  '.pf-m-disabled[data-test-action="Delete Receiver"]',
```

Reply:
that didn't work, had to specify the full `.pf-c-dropdown__menu-item.pf-m-disabled[data-test-action="Delete Receiver"]`